src/global.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@styled-system/prop-types";
```

Comment:
We shouldn't need this, what was your reason for adding it?

Reply:
There was a lint error, I'll see what happens when it's removed

Reply:
Yeah we don't need it not sure why I added it tbh

Reply:
It doesn't look like there's any type defined in that package and I can't find any in [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)

Reply:
But that package is just for js prop types isn't it?

Reply:
Ah I see, we're actually using this in the `filter-out-styled-system-spacing-props` util. 

My thought was this package only contains js prop-types so it won't be used in any TypeScript files, but I see that we are. This is fine for now imo, but by the end of the ts conversion that will be the only usage of `@styled-system/prop-types` so I wonder if in the future you'll want to find another way of writing that util without it so the dependency can be removed.

Reply:
Yeah I agree we should look to replace it with something else down the line